docs/articles/dockerfile_best-practices.md

```diff
@@ -153,8 +153,10 @@ maintainers. If you know there’s a particular package, `foo`, that needs to be
 updated, use `apt-get install -y foo` and it will update automatically.
 
 * Do write instructions like:
-
-    RUN apt-get update && apt-get install -y package-bar package-foo package-baz
+    RUN apt-get update && apt-get install -y \
```

Comment:
If you add a *blank line* before this line (to separate the code-block from the bullet), and indent the whole block with an additional 4 spaces (so, the first line is indented with 8 spaces), it should render correctly.

I just cloned your branch, made those changes and ran `make docs`. It then renders correctly;

![14571-screenshot](https://cloud.githubusercontent.com/assets/1804568/8668100/c1673f5c-2a07-11e5-9a87-7d10375d70ed.jpg)